docs/dictionary/control_st/if.lcdoc

```diff
@@ -4,9 +4,25 @@ Type: control structure
 
 Syntax: if <condition> then <statementList> [else <elseStatementList>]
 
+Syntax:if <condition> then
+ <statementList>
+ [else if <condition> then
+   <elseifStatementList>]
+ [else 
+   <elseStatementList>]
+end if
+
+
 Summary:
-<execute|Executes> a list of <statement|statements> if a condition is
-true. 
+A <conditional statement> following an <if> or an optional series of
```

Comment:
@MartinKoob wondering if this could be reworded so we don't have `conditional statement` so many times in a sentence. Also I think `<conditional expression|condition>` would be more accurate here. Probably the optional `else if` could be moved to a different sentence.